docs/src/pages/docs/api.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 name: API
 title: API
-route: /docs/api
+route: /docs/rest-api
```

Comment:
This is the hack/workaround, I was getting collision warnings on `/docs/api` vs `/docs/api/`.